### DISCUSSION TITLE
Add New Dynamic tags in Elementor Pro

### DISCUSSION THREAD

**Discussion by ayoubkhan558** [elementor/elementor#21541] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Elementor lacks some much-needed dynamics.
Currently, Elementor provides dynamic tags for post title, post author, post date, and some others, but it lacks tags for post content and post category. I believe this feature would be useful for many users who want to create customized templates for their posts or display specific information related to posts on their website.

### Describe the solution you'd like

It would be great if you could add new dynamic tags for 

- post content
- post category
- Post Content
- Post Parent
- Post Status
- Post Type
- IsSticky
- Comments Count

YES/NO TAGS
- Are Comments allowed
- Current User Can (can check, for example if user can edit_posts)
- Is author of post
- Is feed
- Is Frontpage
- Is Home
- Is Post in category
- Is Post in list
- Is Singular

This would allow users to easily create dynamic templates and display post information on their websites without the need for additional plugins or coding.
It would be great if you could consider adding these dynamic tags in the next release. Thank you for your attention and consideration.

### Describe alternatives you've considered

[Jet](https://crocoblock.com/)
https://www.dynamic.ooo/

**Comment by loucasl** [comment]
Post category is the only one on your list that does actually have a dynamic tag associated, it's just called "post terms"

**Comment by ghost** [comment]
+1

**Comment by ghost** [comment]
Also add dynamic tag to display the number of available posts and product

**Comment by git-temak** [comment]
Post content is more than highly required. Spoken to support and this was their response

> The reason it works differently is due to how WordPress content is structured. Unlike other dynamic tags that return plain text or URLs, Post Content can contain complex HTML, Gutenberg blocks, shortcodes, or even Elementor layouts. Allowing it to be used as a standard dynamic tag could lead to rendering issues or recursive loops in certain scenarios.

Understandably, the post content field could contain different types of tags and info, and rightfully so, but maybe a compromise would be to allow it in some locations and not others. One very common use case is staff profiles on a team page or about page to display staff names and bio. Currently, loop items cannot show post content without finding other means as its not available in dynamic content. This would massively help many users and keep the functionality core to Elementor, rather than requiring them to create extra fields or use additional plugins.
